src/server/utils/watchLocalModules.js

```diff
@@ -14,52 +14,145 @@
  * permissions and limitations under the License.
  */
 
-import fs from 'node:fs';
 import path from 'node:path';
-import chokidar from 'chokidar';
+import fs from 'node:fs/promises';
+import { glob } from 'glob';
 import loadModule from 'holocron/loadModule.node';
 import {
   getModules,
   getModuleMap,
   resetModuleRegistry,
   addHigherOrderComponent,
 } from 'holocron/moduleRegistry';
-import { getIp } from './getIP';
 import onModuleLoad from './onModuleLoad';
 
+const CHANGE_WATCHER_INTERVAL = 1000;
+const WRITING_FINISH_WATCHER_TIMEOUT = 400;
```

Comment:
these should be configurable by the user
I'm debating via [arguments](https://github.com/americanexpress/one-app/blob/main/src/server/config/env/argv.js) or [environment variables](https://github.com/americanexpress/one-app/blob/main/src/server/config/env/runTime.js)

Reply:
perhaps we let that be another PR so as to get this fix in, and then resolve the plan?

Reply:
I'd prefer arguments. Particularly since this is just local dev. They are much more discoverable since they are self documenting. Also those using one-app-runner will be able to very easily include them in the runner config.